snapcraft/plugins/make.py

```diff
@@ -83,12 +83,13 @@ def schema(cls):
             'default': [],
         }
 
+        return schema
+
+    @classmethod
+    def get_build_properties(cls):
         # Inform Snapcraft of the properties associated with building. If these
         # change in the YAML Snapcraft will consider the build step dirty.
-        schema['build-properties'].extend(
-            ['makefile', 'make-parameters', 'make-install-var'])
-
-        return schema
+        return ['makefile', 'make-parameters', 'make-install-var', 'artifacts']
```

Comment:
Yes, I am aware of having to fix cmake.  It will hit a PR once possible to fix w/ this code base.
cmake already uses get_build_properites(), so I'll have to call MakePlugin.get_buid_properties() to fix, but can't until this PR, I think....